nautobot/dcim/tests/test_views.py

```diff
@@ -1439,13 +1910,53 @@ def setUpTestData(cls):
         cls.bulk_create_data = {
             "device_type": devicetypes[1].pk,
             "name_pattern": "Device Bay Template [4-6]",
+            "description": "View Test Bulk Create Device Bays",
         }
 
         cls.bulk_edit_data = {
             "description": "Foo bar",
         }
 
 
+class ModuleBayTemplateTestCase(ViewTestCases.ModularDeviceComponentTemplateViewTestCase):
+    model = ModuleBayTemplate
+    rename_field = "position"
+
+    @classmethod
+    def setUpTestData(cls):
+        device_type = DeviceType.objects.first()
+        module_type = ModuleType.objects.first()
+
+        cls.form_data = {
+            "device_type": device_type.pk,
+            "module_type": None,
+            "position": "Module Bay Template X",
+            "description": "Test modulebaytemplate description",
+            "label": "Test modulebaytemplate label",
+        }
+
+        cls.bulk_create_data = {
+            "module_type": module_type.pk,
+            "position_pattern": "Test Module Bay Template [5-7]",
+            "label_pattern": "Test modulebaytemplate label [1-3]",
+            "description": "Test modulebaytemplate description",
+        }
+
+        cls.bulk_edit_data = {
+            "description": "Description changed",
+        }
+
+    def _edit_object_test_setup(self):
+        test_instance = self._get_queryset().first()
+        self.update_data = {
+            "position": test_instance.position,
+            "device_type": getattr(getattr(test_instance, "device_type", None), "pk", None),
+            "module_type": getattr(getattr(test_instance, "module_type", None), "pk", None),
+            "label": "new test label",
+            "description": "new test description",
+        }
```

Comment:
Why can't we do this as part of `setUpTestData`?

Reply:
It looks like we can